tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es5",
+        "noImplicitAny": false,
+        "sourceMap": false,
+        "outDir": "dist"
+    }
```

Comment:
Since this is already using TypeScript 2.0, I'd suggest using `"strictNullChecks": true`.